virt/install/installing-ove.adoc

```diff
@@ -0,0 +1,52 @@
+:_mod-docs-content-type: ASSEMBLY
+include::_attributes/common-attributes.adoc[]
+[id="installing-ove"]
+= Installing an {ove} cluster
+:context: installing-ove
+
+toc::[]
+
+The following procedures describe a method to install a {ove-first} cluster that uses a simplified user interface, and does not require you to set up an external image registry in disconnected environments, which is a requirement for all other disconnected installation methods.
```

Comment:
Perhaps make this two sentences by changing _, and does not require_  to _. This method does not require_